Dockerfile

```diff
@@ -18,7 +18,7 @@ FROM ${BASE_CONTAINER} as builder
 ARG TARGETPLATFORM
 
 # Update pip and setuptools
-RUN pip install --upgrade pip setuptools  
+RUN pip install "pip==23.2.1" "setuptools==68.2.2"
```

Comment:
Curious why fixing these versions?

Reply:
I think its good to fix them eitherways, plus it seems like these versions are required for mpi4py